scipy/io/mmio.py

```diff
@@ -30,7 +33,7 @@ def mminfo(source):
     Parameters
     ----------
 
-    source : file
+    filename : file
```

Comment:
Agreed, although, for the first argument, it would be rare that someone used it as a keyword argument.